<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>690</width>
         <height>1100</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Open files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Replace</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="text_original_1">
              <property name="text">
               <string>#proiectant_rez</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_1">
              <property name="text">
               <string>S.C.GEOMAR PROIECT SRL</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="text_original_2">
              <property name="text">
               <string>#proiectant_rez_si_arh</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_2">
              <property name="text">
               <string>S.C. GEOMAR PROIECT, C. ARH. A. HEROIU</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="text_original_3">
              <property name="text">
               <string>#faza</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_3">
              <property name="text">
               <string>D.T.A.C.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="text_original_4">
              <property name="text">
               <string>#data</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_4">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="text_original_5">
              <property name="text">
               <string>#beneficiar_nume</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_5">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLineEdit" name="text_original_6">
              <property name="text">
               <string>#beneficiar_adresa</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_6">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="text_original_7">
              <property name="text">
               <string>#beneficiar_telefon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_7">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLineEdit" name="text_original_8">
              <property name="text">
               <string>#beneficiar_email</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_8">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLineEdit" name="text_original_9">
              <property name="text">
               <string>#obiectiv</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_9">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLineEdit" name="text_original_10">
              <property name="text">
               <string>#forma_de_detinere_teren</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_10">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLineEdit" name="text_original_11">
              <property name="text">
               <string>#adresa_imobil</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_11">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLineEdit" name="text_original_12">
              <property name="text">
               <string>#acces_imobil</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_12">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLineEdit" name="text_original_13">
              <property name="text">
               <string>#nr_cadastral</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_13">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLineEdit" name="text_original_14">
              <property name="text">
               <string>#vecin_n</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_14">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLineEdit" name="text_original_15">
              <property name="text">
               <string>#vecin_e</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_15">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLineEdit" name="text_original_16">
              <property name="text">
               <string>#vecin_s</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_16">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLineEdit" name="text_original_17">
              <property name="text">
               <string>#vecin_v</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_17">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLineEdit" name="text_original_18">
              <property name="text">
               <string>#suprafata_teren</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_18">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLineEdit" name="text_original_19">
              <property name="text">
               <string>#suprafata_construita_existenta</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_19">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLineEdit" name="text_original_20">
              <property name="text">
               <string>#suprafata_desfasurata_existenta</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_20">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLineEdit" name="text_original_21">
              <property name="text">
               <string>#POT_existent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_21">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QLineEdit" name="text_original_22">
              <property name="text">
               <string>#CUT_existent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_22">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <item>
             <widget class="QLineEdit" name="text_original_23">
              <property name="text">
               <string>#suprafata_construita_desfiintata</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_23">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_24">
            <item>
             <widget class="QLineEdit" name="text_original_24">
              <property name="text">
               <string>#suprafata_desfasurata_desfiintata</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_24">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <item>
             <widget class="QLineEdit" name="text_original_25">
              <property name="text">
               <string>#suprafata_construita_propusa</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_25">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_26">
            <item>
             <widget class="QLineEdit" name="text_original_26">
              <property name="text">
               <string>#suprafata_desfasurata_propusa</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_26">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_27">
            <item>
             <widget class="QLineEdit" name="text_original_27">
              <property name="text">
               <string>#suprafata_construita_totala</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_27">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_28">
            <item>
             <widget class="QLineEdit" name="text_original_28">
              <property name="text">
               <string>#suprafata_desfasurata_totala</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_28">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_29">
            <item>
             <widget class="QLineEdit" name="text_original_29">
              <property name="text">
               <string>#POT_propus</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_29">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_30">
            <item>
             <widget class="QLineEdit" name="text_original_30">
              <property name="text">
               <string>#CUT_propus</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="text_current_30">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
